services/blockchain-indexer/shared/chain.js

```diff
@@ -23,7 +23,7 @@ const isMainchain = async () => {
 		const networkStatus = (await dataService.getNetworkStatus()).data;
 		chainID = networkStatus.chainID;
 	}
-	return chainID.match(regex.MAINCHAIN_ID);
+	return regex.MAINCHAIN_ID.test(chainID);
```

Comment:
Could you please verify if we use `String.match()` anywhere else within the codebase instead of `regex.test()`?